### DISCUSSION TITLE
Support for 33cm boards?

### DISCUSSION THREAD

**Discussion by redrob** [lora-aprs/LoRa_APRS_iGate#143] [Q&A]
I realize this wouldn't apply to the European hams, but here in NA we have access to the 33cm. Many of us have been experimenting with LoRa / Meshtastic on 33cm, so we already have 33cm boards. Does the code support the 915 MHz boards such as the T-Beam - M8N & SX1262 and T-Beam 1.1 ?

**Comment by peterus** [comment]
I am not sure about the SX1262. I do not know this LoRa chip.
If it has SPI it could work 😉

In theory you just have to change the frequencies in the config file. The rest is working out of the box.

**Comment by Superuser998** [comment]
I have the T-Beam SX1276 and was able to use this project by changing the frequencies in the config file an changing the radio in line 28 of TaskRadioLib.cpp.

  radio  = new SX1276(module); // changed from SX1278 to SX1276 for 915mhz T beam
